evalml/pipelines/components/estimators/regressors/arima_regressor.py

```diff
@@ -170,6 +174,7 @@ def fit(self, X, y=None):
             ValueError: If y was not passed in.
         """
         if X is not None:
+            X = downcast_int_nullable_to_double(X)
             X = X.fillna(X.mean())
         X, y = self._manage_woodwork(X, y)
```

Comment:
Yes this call was added to work with `fillna` but also more generally because `ARIMA` won't accept `Int64` dtypes

Reply:
Gotcha. It may be safer to move the call to `manage_woodwork` above this regardless, if I'm not missing something.

Reply:
I'm hesitant about moving it to `manage_woodwork` because that affects every instance where it's called. But I believe we're in the process of unifying nullable and non-nullable types so this won't be an issue soon

Reply:
Sorry sorry, I just meant calling `_manage_woodwork` before calling `downcast_int_nullable_to_double` within this file, not within the `_manage_woodwork` function itself! Apologies if that was unclear.